src/modules/coreHttpApi/common/BaseController.ts

```diff
@@ -1,9 +1,12 @@
 import { Result } from "@js-soft/ts-utils";
 import express from "express";
-import { Return } from "typescript-rest";
+import { Context, Return, ServiceContext } from "typescript-rest";
 import { Envelope } from "../../../infrastructure";
 
 export abstract class BaseController {
+    @Context
+    protected context: ServiceContext;
```

Comment:
where do you use this?

Reply:
Currently, not used in code. Only for debugging to always have the context without the need to add it as a parameter.

Later it is planned to remove the context parameter from all functions and use this member

Reply:
please remove it until you actually use it. Additionally i'm not a big fan of this top level context as it is not as self explanatory as the context that is injected in the methods